docs/other/fontsobject.rst

```diff
@@ -107,7 +122,7 @@ StyleName               A string containing the Style Name of the font.
 
 **Returns**
 
-An array of fontObjects.
+An array of :ref:`fontobject`.
```

Comment:
Prefer the use of labels for references so the sentences can read correctly while still linking to the Font object entry.
```suggestion
Array of :ref:`Font objects<fontobject>`; read-only.
```